<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cave</class>
 <widget class="QWidget" name="Cave">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="cave_field" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>500</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_gen_mode">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>50</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	background-color: rgba(0, 0, 0, 0.0);
	color: rgba(255, 255, 255, 0.7);
	border: 1px solid rgba(255, 255, 255, 0);
	selection-background-color: rgba(255, 255, 255, 30);
}

QComboBox::disabled {
	background-color: rgba(0, 0, 0, 0.0);
	color: rgba(100, 100, 100, 0.7);
	border: 1px solid rgba(100, 100, 100, 0);
	selection-background-color: rgba(255, 255, 255, 30);
}</string>
   </property>
   <item>
    <property name="text">
     <string>Random</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>File</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="d_init_chance">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>140</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	background-color: rgba(0, 0, 0, 0);
	alternate-background-color: rgba(0, 0, 0, 0);
	color: rgb(255, 255, 255);
	border-radius: 10px;
	border: 1px solid rgb(34, 122, 255);
}

QDoubleSpinBox:disabled {
	background-color: rgba(0, 0, 0, 0);
	alternate-background-color: rgba(0, 0, 0, 0);
	color: rgb(143,152,160);
	border-radius: 10px;
	border: 1px solid #53616F;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>50.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reset">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>520</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgb(230, 230, 230);
	background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #07BEFF stop: 1 #2B74FF);
	alternate-background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #07BEFF stop: 1 #2B74FF);
	border: 1px solid rgba(255, 255, 255, 0);
	border-radius: 10px;
}

QPushButton:hover {
    color: rgb(230, 230, 230);
	background-color: qlineargradient(x1: 0.4, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
	alternate-background-color: qlineargradient(x1: 0.4, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
	border: 1px solid rgba(255, 255, 255, 0);
	border-radius: 10px;
}

QPushButton:pressed {
	color: rgb(254, 253, 255);
    background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
  alternate-background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
  border-radius: 10px;
}

QPushButton:disabled {
	color: rgb(143,152,160);
	background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #2C3C49 stop: 1 #53616F);
	alternate-background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #2C3C49 stop: 1 #53616F);
	border: 1px solid rgba(0, 0, 0, 0);
	border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="i_birth_limit">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>240</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
	background-color: rgba(0, 0, 0, 0);
	alternate-background-color: rgba(0, 0, 0, 0);
	color: rgb(255, 255, 255);
	border-radius: 10px;
	border: 1px solid rgb(34, 122, 255);
}

QSpinBox:disabled {
	background-color: rgba(0, 0, 0, 0);
	alternate-background-color: rgba(0, 0, 0, 0);
	color: rgb(143,152,160);
	border-radius: 10px;
	border: 1px solid #53616F;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <number>7</number>
   </property>
   <property name="value">
    <number>4</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="i_death_limit">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>290</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
	background-color: rgba(0, 0, 0, 0);
	alternate-background-color: rgba(0, 0, 0, 0);
	color: rgb(255, 255, 255);
	border-radius: 10px;
	border: 1px solid rgb(34, 122, 255);
}

QSpinBox:disabled {
	background-color: rgba(0, 0, 0, 0);
	alternate-background-color: rgba(0, 0, 0, 0);
	color: rgb(143,152,160);
	border-radius: 10px;
	border: 1px solid #53616F;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <number>7</number>
   </property>
   <property name="value">
    <number>4</number>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_render_mode">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>350</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	background-color: rgba(0, 0, 0, 0.0);
	color: rgba(255, 255, 255, 0.7);
	border: 1px solid rgba(255, 255, 255, 0);
	selection-background-color: rgba(255, 255, 255, 30);
}

QComboBox::disabled {
	background-color: rgba(0, 0, 0, 0.0);
	color: rgba(100, 100, 100, 0.7);
	border: 1px solid rgba(100, 100, 100, 0);
	selection-background-color: rgba(255, 255, 255, 30);
}</string>
   </property>
   <item>
    <property name="text">
     <string>Manually</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Automatic</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="i_render_delay">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>400</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
	background-color: rgba(0, 0, 0, 0);
	alternate-background-color: rgba(0, 0, 0, 0);
	color: rgb(255, 255, 255);
	border-radius: 10px;
	border: 1px solid rgb(34, 122, 255);
}

QSpinBox:disabled {
	background-color: rgba(0, 0, 0, 0);
	alternate-background-color: rgba(0, 0, 0, 0);
	color: rgb(143,152,160);
	border-radius: 10px;
	border: 1px solid #53616F;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>150</number>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
   <property name="singleStep">
    <number>50</number>
   </property>
   <property name="value">
    <number>500</number>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_border_mode">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>190</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	background-color: rgba(0, 0, 0, 0.0);
	color: rgba(255, 255, 255, 0.7);
	border: 1px solid rgba(255, 255, 255, 0);
	selection-background-color: rgba(255, 255, 255, 30);
}

QComboBox::disabled {
	background-color: rgba(0, 0, 0, 0.0);
	color: rgba(100, 100, 100, 0.7);
	border: 1px solid rgba(100, 100, 100, 0);
	selection-background-color: rgba(255, 255, 255, 30);
}</string>
   </property>
   <item>
    <property name="text">
     <string>No border</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Border</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_render">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>450</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgb(230, 230, 230);
	background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #07BEFF stop: 1 #2B74FF);
	alternate-background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #07BEFF stop: 1 #2B74FF);
	border: 1px solid rgba(255, 255, 255, 0);
	border-radius: 10px;
}

QPushButton:hover {
    color: rgb(230, 230, 230);
	background-color: qlineargradient(x1: 0.4, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
	alternate-background-color: qlineargradient(x1: 0.4, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
	border: 1px solid rgba(255, 255, 255, 0);
	border-radius: 10px;
}

QPushButton:pressed {
	color: rgb(254, 253, 255);
    background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
  alternate-background-color: qlineargradient(x1: 0.3, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #02BFFF stop: 1 #2E73FF);
  border-radius: 10px;
}

QPushButton:disabled {
	color: rgb(143,152,160);
	background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #2C3C49 stop: 1 #53616F);
	alternate-background-color: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5,
                                      stop: 0 #2C3C49 stop: 1 #53616F);
	border: 1px solid rgba(0, 0, 0, 0);
	border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>Render next</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_gen_mode">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>50</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(160, 159, 161);</string>
   </property>
   <property name="text">
    <string>Generating mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_init_chance">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>140</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(160, 159, 161);</string>
   </property>
   <property name="text">
    <string>Cell initialization chance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_birth_limit">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>240</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(160, 159, 161);</string>
   </property>
   <property name="text">
    <string>Cell birth lilmit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_death_limit">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>290</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(160, 159, 161);</string>
   </property>
   <property name="text">
    <string>Cell death lilmit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_render_mode">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>350</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(160, 159, 161);</string>
   </property>
   <property name="text">
    <string>Rendering mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_render_delay">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>400</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(160, 159, 161);</string>
   </property>
   <property name="text">
    <string>Rendering delay</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_size">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>90</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(160, 159, 161);</string>
   </property>
   <property name="text">
    <string>Field Size</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="i_size">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>90</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
	background-color: rgba(0, 0, 0, 0);
	alternate-background-color: rgba(0, 0, 0, 0);
	color: rgb(255, 255, 255);
	border-radius: 10px;
	border: 1px solid rgb(34, 122, 255);
}

QSpinBox:disabled {
	background-color: rgba(0, 0, 0, 0);
	alternate-background-color: rgba(0, 0, 0, 0);
	color: rgb(143,152,160);
	border-radius: 10px;
	border: 1px solid #53616F;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_file_name">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 100, 100);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
